src/Lachain.Console/Application.cs

```diff
@@ -113,6 +134,89 @@ public void Start(RunOptions options)
                 Thread.Sleep(1000);
         }
 
+        private Block BuildNextBlock(IStateManager _stateManager, IPrivateWallet _wallet,
```

Comment:
I don't sure,  but do we need this methods here? I can't find where are they called.)

Reply:
it was redundant code... removed